lib/EntityGenerator/src/Field/AbstractFieldGenerator.php

```diff
@@ -159,6 +160,22 @@ protected function addFieldAttributes(Property $property, PhpNamespace $namespac
             }
         }
 
+        if ($this->field->isRequired() && (!$this->field->isVirtual() || in_array($this->field->getType(), [FieldType::MANY_TO_MANY_T, FieldType::MANY_TO_ONE_T]))) {
+            if ($this->field->isManyToMany()
```

Comment:
Warning: you cannot use CoreBundle classes, enum and other types. Only use `NodeTypeFieldInterface` methods